CONTRIBUTING.md

```diff
@@ -232,3 +232,39 @@ releases will be based off of that release.
 - delete the `pre-rc/vX.X.X` and `RC` branches
 - create a PR into `master` containing ONLY the CHANGELOG.md updates
 - tag and release `release/vX.XX.X`
+
+## Code Owner Membership 
+
+In the ethos of open source projects, and out of necessity to keep the code
+alive, the core contributor team will strive to permit special repo privileges
+to developers who show an aptitude towards developing with this code base. 
+
+Several different kinds of privileges may be granted however most common
+privileges to be granted are merge rights to either part of, or the entire the
+code base (though the github `CODEOWNERS` file). The on-boarding process for
```

Comment:
How about differentiate on a module basis (module owner), and the final power is repo owner which can make and merge changes anywhere

Reply:
I think the existing description encompasses the possibility of the module owners, as a module is "a part of" the code base. Leaving as is